backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -208,6 +208,11 @@ bool SdlGraphicsManager::notifyMousePosition(Common::Point &mouse) {
 	mouse.y = CLIP<int16>(mouse.y, 0, _windowHeight - 1);
 
 	int showCursor = SDL_DISABLE;
+	// DPI aware scaling to mouse position
```

Comment:
Wouldn't it be better to use a float here, too - you might get rounding issues in the higher coordinate ranges otherwise.

Reply:
I am not sure what you mean here, `scale` could be 1 or 2 only.